webclient/src/Hacksite.vue

```diff
@@ -35,4 +39,18 @@ hr {
   border: none;
   border-bottom: 1px solid #000;
 }
+.full-screen-modal {
+  height: 100%;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  background: rgba(255, 255, 255, .8)
```

Comment:
can we switch this to `rgba(0, 0, 0, .25)` -- looks a little better than whitewashing everything